pom.xml

```diff
@@ -56,6 +57,28 @@
 			<artifactId>spring-security-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
`spring-boot-starter-data-jpa` includes all required dependencies for hibernate and jpa, so it's redundant.